api/src/pcapi/routes/backoffice/finance/finance_incidents_blueprint.py

```diff
@@ -800,17 +800,24 @@ def get_finance_incident_validation_form(finance_incident_id: int) -> utils.Back
 
 
 @finance_incidents_blueprint.route("/<int:finance_incident_id>/validate", methods=["POST"])
-@utils.permission_required(perm_models.Permissions.MANAGE_INCIDENTS)
+@utils.permission_required_in(
+    (perm_models.Permissions.MANAGE_INCIDENTS, perm_models.Permissions.VALIDATE_COMMERCIAL_GESTURE)
+)
 def validate_finance_incident(finance_incident_id: int) -> utils.BackofficeResponse:
     finance_incident = _get_incident(finance_incident_id)
     form = forms.IncidentValidationForm()
+    error_message = None
+
+    if finance_incident.kind == finance_models.IncidentType.COMMERCIAL_GESTURE:
+        utils.check_any_permission_of((perm_models.Permissions.VALIDATE_COMMERCIAL_GESTURE,))
```

Comment:
OK, mais dans l'autre sens, a-t-on le droit de valider les autres types d'incident avec la seule permission `VALIDATE_COMMERCIAL_GESTURE` ?

Si non, j'imagine : 
```suggestion
    if finance_incident.kind == finance_models.IncidentType.COMMERCIAL_GESTURE:
        utils.check_any_permission_of((perm_models.Permissions.VALIDATE_COMMERCIAL_GESTURE,))
    else:
        utils.check_any_permission_of((perm_models.Permissions.MANAGE_INCIDENTS,))
```

Reply:
ok deal, je bouge ça dans une autre view